File: src/Taller1/ej3/MultipleCounter.java
package Taller1.ej3;

import Taller1.ej2.Counter;

public class MultipleCounter extends Counter{
    private int delta;
    public MultipleCounter(int delta){
        super();
        this.delta = delta;
    }

    public void increment(){
        for(int i = 0; i < delta; i++){
            super.increment();
        }
    }

    public void decrement(){
        for(int i = 0; i < delta; i++){
            super.decrement();
        }
    }
}

File: src/Taller10/ej4/multiple_counter.rb
# frozen_string_literal: true

require_relative 'counter'
class MultipleCounter < Counter
    def initialize(delta)
        super()
        @delta = delta
    end
    def increment
        aux = @delta
        while aux > 0
            super
            aux -= 1
        end
    end
    def decrement
        aux = @delta
        while aux > 0
            super
            aux -= 1
        end
    end
end
